<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMTrainDataWidget</class>
 <widget class="QWidget" name="QMTrainDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schulungen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mitarbeiter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbFilterTrain">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Datum ab</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Schulung</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="cbFilterEmployee">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cbFilterState">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxVisibleItems">
        <number>20</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QToolButton" name="tbFilterReset">
       <property name="toolTip">
        <string>Filter zurücksetzen</string>
       </property>
       <property name="text">
        <string>Zurücksetzen</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/edit-reset.svg</normaloff>:/icons/icons/edit-reset.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QToolButton" name="tbFilter">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Daten filtern</string>
       </property>
       <property name="text">
        <string>Filtern</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/view-filter.svg</normaloff>:/icons/icons/view-filter.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Datum bis</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDateEdit" name="deDateFilterFrom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDateEdit" name="deDateFilterTo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,0">
     <item>
      <widget class="QTableView" name="tvTrainData">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QDockWidget" name="dwTrainDataDetails">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>565</height>
        </size>
       </property>
       <property name="features">
        <set>QDockWidget::DockWidgetClosable</set>
       </property>
       <property name="windowTitle">
        <string>Details</string>
       </property>
       <widget class="QWidget" name="dockWidgetContents">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Mitarbeiter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbEmployee">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Schulung</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbTraining">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Datum</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCalendarWidget" name="cwDate"/>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Status</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbState">
             <property name="insertPolicy">
              <enum>QComboBox::NoInsert</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Nachweise</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="tbRemoveCertificate">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbAddCertificate">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../icons.qrc">
                 <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tvCertificates">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="tbAddSingle">
       <property name="toolTip">
        <string>Neuen Eintrag hinzufügen</string>
       </property>
       <property name="text">
        <string>Eintrag hinzufügen</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbDeleteSelected">
       <property name="toolTip">
        <string>Ausgewählt Einträge löschen</string>
       </property>
       <property name="text">
        <string>Ausgewählte löschen</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbTrainDataDetails">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/view-list-details.svg</normaloff>:/icons/icons/view-list-details.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbImportCsv">
       <property name="toolTip">
        <string>Import von Daten aus einer CSV-Datei</string>
       </property>
       <property name="text">
        <string>CSV importieren</string>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/icons/icons/text-csv.svg</normaloff>:/icons/icons/text-csv.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>tbFilterReset</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainDataWidget</receiver>
   <slot>resetFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>799</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbFilter</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainDataWidget</receiver>
   <slot>updateFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>837</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbAddSingle</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainDataWidget</receiver>
   <slot>addSingleEntry()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbDeleteSelected</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainDataWidget</receiver>
   <slot>deleteSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbTrainDataDetails</sender>
   <signal>clicked()</signal>
   <receiver>QMTrainDataWidget</receiver>
   <slot>showTrainDataDetails()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetFilter()</slot>
  <slot>addSingleEntry()</slot>
  <slot>updateFilter()</slot>
  <slot>deleteSelected()</slot>
  <slot>importCsv()</slot>
  <slot>showTrainDataDetails()</slot>
 </slots>
</ui>
